intro/scipy/index.rst

```diff
@@ -157,73 +157,72 @@ functions here. Frequently used ones are:
 Linear algebra operations: :mod:`scipy.linalg`
 ----------------------------------------------
 
-.. tip::
```

Comment:
"Tip" is all over the place. Do you agree that this (and many of them) are unnecessary?